frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
Any particular reason for the `JSON.parse(JSON.stringify(...))`? I removed that and just pass the `geojson` object to `geojsonExtent`.